docs/spark/spark-writes.md

```diff
@@ -311,7 +311,9 @@ distribution & sort order to Spark.
 {{< /hint >}}
 
 {{< hint info >}}
-Both global sort (`orderBy`/`sort`) and local sort (`sortWithinPartitions`) work for the requirement.
+With SQL, you can use [`ORDER BY`](https://spark.apache.org/docs/latest/sql-ref-syntax-qry-select-orderby.html) to achieve global sorting or [`SORT BY`](https://spark.apache.org/docs/latest/sql-ref-syntax-qry-select-sortby.html) to achieve local sorting.
```

Comment:
Global and Local sorting need to be defined, I don't think we have it in this part of the doc but I believe we could just refer to it in the distribution section?

Reply:
We are trying to refrain from using pronouns in the documentation

"With SQL, you can use" -> "In SQL the keyword [`ORDER BY`] ..... "